incubator/hnc/config/default/manager_auth_proxy_patch.yaml

```diff
@@ -8,6 +8,10 @@ metadata:
 spec:
   template:
     spec:
+      securityContext:
+        fsGroup: 2000
+        runAsNonRoot: true
+        runAsUser: 1000
```

Comment:
Can you comment where these values came from?

Reply:
Added!